src/go/wsl-helper/cmd/dnsmonitor.go

```diff
@@ -0,0 +1,47 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/rancher-sandbox/rancher-desktop/src/wsl-helper/pkg/dnsmonitor"
+)
+
+// dnsmonitorCmd represents the dnsmonitor command
+var dnsmonitorCmd = &cobra.Command{
+	Use:   "dns-monitor",
```

Comment:
This command should be renamed; we're no longer monitoring anything (it's a one-shot).

Reply:
Should I rename the whole package too? since we decided to call from the app the name might no longer be applicable, @mook-as  WDYT?

Reply:
Well, yes; as it is now it shouldn't be `dnsmonitor`, and something closer to `update-dns` (with the package name something like `resolveupdater` or `dnsupdater` maybe)?